profileservice/api/utils/jsonutils.py

```diff
@@ -49,6 +49,16 @@ def create_log_json(ep_name, ep_method, in_json):
 
     return in_json
 
+def create_simple_pii_log_json(in_json):
```

Comment:
I think it's better to use the same method signature as that of `create_log_json`. That way we don't have to call `create_log_json` followed by `create_simple_pii_log_json` and thus we can reduce one method call, right?

Reply:
I see. I just want to leave the old one, just in case if we want to use it again. I will change it.

Reply:
modified

Reply:
Thanks.